src/db/Kpi/props.js

```diff
@@ -11,6 +11,10 @@ export default {
     type: 'string',
     required: false,
   },
+  accrualPeriodicity: {
```

Comment:
Det var da et forferdelig vanskelig ord. :p Hva med `updateFrequency` eller noe i den duren?

Reply:
Tanken var å gjenbruke etablert vokabular, men har kanskje bare begrenset mening i denne konteksten, så kan bare droppe det.

Reply:
Ja, i alle fall for de interne variablene våre, for det variabelnavnet var ikke lett å forstå uten å slå det opp... :p